### PR TITLE
Powercrovel

### PR DESCRIPTION
**Pull request by Xaytan**
## Adds the POWERCROVEL
<details>
<summary>
Adds a craftable gas-powered crovel, because batteries are lame and gas crovel goes vroom vroom when it kills spiders. 

Awesome sprites courtesy of Dromkii! 
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl: Xaytan
add: Added the POWERCROVEL and crafting recipe
fix: Fixed spelling error preventing shovels from receiving their effective faction damage
fix: Fixed tools with on/off states using fuel while turned off.
fix: Fixed tools with on/off states using cell power while turned off
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/tools/_tools.dm [hunk 1]
```diff
@@ -834,15 +834,15 @@
 		adjustToolHealth(-degradation, user)
 
 //Power and fuel drain, sparks spawn
-/obj/item/proc/check_tool_effects(mob/living/user, time)
+/obj/item/tool/proc/check_tool_effects(mob/living/user, time)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/_tools.dm:837]: please dont do this

File: code/game/objects/items/weapons/tools/_tools.dm [hunk 2]
```diff
@@ -863,10 +863,13 @@
 	return TRUE
 
 //Returns the amount of fuel in tool
-/obj/item/proc/get_fuel()
+/obj/item/tool/proc/get_fuel()
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/_tools.dm:866]: REALLY dont do this one

Comment (Xaytan) [reply code/game/objects/items/weapons/tools/_tools.dm:866]: Neither of those are used on anything that isn't a tool, unless ctrl-shift-f lied to me.

Comment (Xaytan) [reply code/game/objects/items/weapons/tools/_tools.dm:866]: Actually no, I'm dumb. I'll fix that, there's a better way.

**Pull request merged**